docs/CHANGELOG.asciidoc

```diff
@@ -85,6 +85,40 @@ include::upgrade-notes.asciidoc[]
 
 For information about the {kib} 8.16.0 release, review the following information.
 
+The 8.16.0 release includes the following known issues.
+
+[float]
+[[known-issues-8.16.0]]
+=== Known issues
+
+[discrete]
+[[known-199902]]
+.Stack Monitoring shows "Unable to load page error"
+[%collapsible]
+====
+*Details* +
+The Stack Monitoring pages Overview, Nodes, Logs can stop working with the error "Unable to load page error". The Stack trace mentions `TypeError: Cannot read properties of undefined (reading 'logsLocator')`.
+
+*Workaround* +
+None at the moment.
+
+For more information, refer to {kibana-issue}199902[#199902].
+====
+
+[discrete]
+[[known-199891-199892]]
+.Onboarding, tutorial of APM and OpenTelemetry and some "Beats Only" integrations will show the error "Unable to load page error"
+[%collapsible]
+====
+*Details* +
+Tutorials linked from the {kib} home page show an error "Unable to load page error". The Stack trace mentions `The above error occurred in tutorial_TutorialUi`.
+
+*Workaround* +
+None at the moment.
```

Comment:
```suggestion
The APM / OpenTelemetry tutorials represented a shortcut to get important parameters to use in the configuration files quickly.
It is still possible to obtain the same parameters following the documentation tutorials of APM.
```

We should add a link to https://www.elastic.co/guide/en/observability/current/apm-collect-application-data.html and other possible doc references on how to setup apikey / secret token and the APM endpoint of APM Server/Integration.

Reply:
We should also add a link to:
https://www.elastic.co/guide/en/observability/current/get-started-with-fleet-apm-server.html

Reply:
My understanding is that a very common thing for users is to use this page to obtain an API key.

They can also do this by going to `Observability > APM > Settings > Agent keys`. Should we explicitly mention this here if it's a common use case?